2025-10-13 11:03:05,737 INFO    MainThread:5304 [wandb_init.py:setup_run_log_directory():705] Logging user logs to /home/patryk/Desktop/AiR_ISS/sem2/DSN/wandb/run-20251013_110305-vb7odqgo/logs/debug.log
2025-10-13 11:03:05,737 INFO    MainThread:5304 [wandb_init.py:setup_run_log_directory():706] Logging internal logs to /home/patryk/Desktop/AiR_ISS/sem2/DSN/wandb/run-20251013_110305-vb7odqgo/logs/debug-internal.log
2025-10-13 11:03:05,737 INFO    MainThread:5304 [wandb_init.py:init():832] calling init triggers
2025-10-13 11:03:05,737 INFO    MainThread:5304 [wandb_init.py:init():837] wandb.init called with sweep_config: {}
config: {'_wandb': {}}
2025-10-13 11:03:07,065 INFO    MainThread:5304 [wandb_init.py:init():880] starting backend
2025-10-13 11:03:07,065 INFO    MainThread:5304 [wandb_init.py:init():883] sending inform_init request
2025-10-13 11:03:07,068 INFO    MainThread:5304 [wandb_init.py:init():891] backend started and connected
2025-10-13 11:03:07,070 INFO    MainThread:5304 [wandb_run.py:_label_probe_notebook():1330] probe notebook
2025-10-13 11:03:07,070 INFO    MainThread:5304 [wandb_run.py:_label_probe_notebook():1340] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2025-10-13 11:03:07,070 INFO    MainThread:5304 [wandb_init.py:init():961] updated telemetry
2025-10-13 11:03:07,078 INFO    MainThread:5304 [wandb_init.py:init():985] communicating run to backend with 90.0 second timeout
2025-10-13 11:03:07,928 INFO    MainThread:5304 [wandb_init.py:init():1036] starting run threads in backend
2025-10-13 11:03:08,104 INFO    MainThread:5304 [wandb_run.py:_console_start():2506] atexit reg
2025-10-13 11:03:08,104 INFO    MainThread:5304 [wandb_run.py:_redirect():2354] redirect: wrap_raw
2025-10-13 11:03:08,105 INFO    MainThread:5304 [wandb_run.py:_redirect():2423] Wrapping output streams.
2025-10-13 11:03:08,105 INFO    MainThread:5304 [wandb_run.py:_redirect():2446] Redirects installed.
2025-10-13 11:03:08,106 INFO    MainThread:5304 [wandb_init.py:init():1076] run started, returning control to user process
2025-10-13 11:08:13,953 INFO    MainThread:5304 [wandb_run.py:_finish():2272] finishing run deep-neural-network-course/lab4-transformer/vb7odqgo
2025-10-13 11:08:13,954 INFO    MainThread:5304 [wandb_run.py:_atexit_cleanup():2471] got exitcode: 0
2025-10-13 11:08:13,954 INFO    MainThread:5304 [wandb_run.py:_restore():2453] restore
2025-10-13 11:08:13,955 INFO    MainThread:5304 [wandb_run.py:_restore():2459] restore done
2025-10-13 11:08:15,232 INFO    MainThread:5304 [wandb_run.py:_footer_sync_info():3869] logging synced files
